Header header
geometry_msgs/Point left_point
geometry_msgs/Point right_point
geometry_msgs/Point center_point